docs/developer/guidelines/style.md

```diff
@@ -46,4 +46,51 @@ A `flake8` plugin for ensuring a consistent logging format. We enable:
 
 ## [Mypy][mypy-github]
 
-A type checker allowing a mix of dynamic and static typing. This is optional for now.
+A comment-based type checker allowing a mix of dynamic and static typing. This is optional for now. In order to enable `mypy` for a specific integration, open its `tox.ini` file and add the 2 lines in the correct section:
+
+```
+[testenv]
+dd_check_types = true
+dd_mypy_args = <FLAGS> datadog_checks/ tests/
+...
+```
+
+The `dd_mypy_args` can be used to add or overwrite some flags to mypy for this specific integration. Here are some useful arguments you can add:
+
+- `--py2`: If the integration needs to be Python2.7 compatible.
```

Comment:
I'd reword this slightly because the `--py2` is really a *must* for integrations that need to support Python 2 (i.e. most integrations currently).

Maybe we can show it in the code block instead?

```ini
[testenv]
dd_check_types = true
dd_mypy_args = <FLAGS> --py2 datadog_checks/ tests/
...
```